### PR TITLE
chore(ci): fix workflow results parsing

### PR DESCRIPTION
**Pull request by soonum**
This is done to fix benchmarks results stored in database.
Some FFT64 were referenced as FFT128 due to errors during parsing in benchmark workflow.

### CODE REVIEW

File: .github/actionlint.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+self-hosted-runner:
+  # Labels of self-hosted runner in array of strings.
+  labels:
+    - m1mac
```
Comment (IceTDrinker) [inline .github/actionlint.yaml:4]: ?

**Review by IceTDrinker — COMMENTED** [review]
So if I understand correctly f128 benchmarks were run and the f64 as well ?

**Comment by soonum** [comment]
> So if I understand correctly f128 benchmarks were run and the f64 as well ?

Yes, adding the cargo feature `fft128` was only additive so both f64 and f128 were run.

File: .github/actionlint.yaml (hunk 1 above)
Comment (soonum) [reply .github/actionlint.yaml:4]: But I guess this file can be removed since we don't check workflows with actionlint in this repository.

Comment (IceTDrinker) [reply .github/actionlint.yaml:4]: hmm this should be run on an ubuntu-latest don't you think ?

Comment (IceTDrinker) [reply .github/actionlint.yaml:4]: wait that's not how you setup the actionlint stuff before was it ?

Comment (IceTDrinker) [reply .github/actionlint.yaml:4]: ok nevermind don't know what I read here, sorry brain fart

**Review by IceTDrinker — APPROVED** [review]

**Pull request merged**